vllm/multimodal/utils.py

```diff
@@ -53,14 +52,10 @@ async def fetch_image(cls, image_url: str) -> Image.Image:
                 "Invalid 'image_url': A valid 'image_url' must start "
                 "with either 'data:image' or 'http'.")
 
+        image.load()
```

Comment:
Good catch! Did you observed a memory leak from not having this?

Reply:
nope, it will just complain about missing fp. I guess @DarkLight1337 also saw something similar so he added image.load() in ImageData initialization.